README.md

```diff
@@ -0,0 +1,16 @@
+# Accelerator
+Accelerator replaces the default SRCDS crash handler with one that is a lot more reliable and uploads the crash reports to a community-accessible processing backend.
+This is useful since Valve's processing system discards crash reports from modded servers - and they still receive a representative sample of crash reports for actual game issues.
+
+The processing backend ([Throttle](https://crash.limetech.org/)) analyses crash reports to extract useful information, and in the case of common issues affecting many servers where manual analysis has been done, tags them with helpful notices with information on resolving the cause of the crash.
```

Comment:
> Throttle **analyses** crash reports 

Does what now?
Perhaps: `Throttle parses crash reports` or something? *analyses* is the plural form of analysis and just doesn't sound right imo. However I'm open to be proven wrong I was in special education for a few years and English vocab is not my strong suite.